agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -77,7 +77,8 @@ function help {
     echo " -np : if provided, do not enable the Prometheus collector.  Defaults to enabling Prometheus collector"
     echo " -sn : if provided, name of the sysdig instance (optional)"
     echo " -op : if provided, perform the installation using the OpenShift command line"
-    echo " -as : if provided, use agent-slim and agent-kmodule with the daemonset"
+    echo " -as : if provided, use agent-slim (this is the default agent). Note: this option is deprecated, please stop using it as it can be removed soon"
```

Comment:
please remove
`, please stop using it as it can be removed soon`

Reply:
@mbreitung @0snug0 Removed